pkg/instancetype/instancetype_test.go

```diff
@@ -1319,7 +1319,6 @@ var _ = Describe("Instancetype and Preferences", func() {
 				Expect(*vmi.Spec.Domain.Devices.AutoattachMemBalloon).To(BeFalse())
 				Expect(*vmi.Spec.Domain.Devices.AutoattachInputDevice).To(BeTrue())
 				Expect(vmi.Spec.Domain.Devices.Disks[0].Cache).To(Equal(v1.CacheWriteBack))
-				Expect(vmi.Spec.Domain.Devices.Disks[0].IO).To(Equal(v1.IODefault))
```

Comment:
Could you also assert that this hasn't changed from `v1.IONative`?

Reply:
Done, and I am willing to open another PR for adding `io_uring` support as well if it is necessary.